.gitignore

```diff
@@ -1,3 +1,9 @@
 .idea/
 *.iml
-.gradle/
\ No newline at end of file
+.gradle/
```

Comment:
shouldn't it all be ignored by default?

Reply:
Adding .gitignore excludes compile files and temporary build artifacts. We need this as part of the commit for everyone.